chainio/clients/wallet/privatekey_wallet.go

```diff
@@ -3,31 +3,30 @@ package wallet
 import (
 	"context"
 	"fmt"
-	"math/big"
 
 	"github.com/Layr-Labs/eigensdk-go/logging"
 	"github.com/Layr-Labs/eigensdk-go/signerv2"
 	"github.com/Layr-Labs/eigensdk-go/utils"
-	"github.com/ethereum/go-ethereum/accounts/abi"
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
-var _ Wallet = (*privateKeyWallet)(nil)
+type EthBackend interface {
```

Comment:
This should be private

Reply:
[de54bde](https://github.com/Layr-Labs/eigensdk-go/pull/284/commits/de54bdee8272c3c1059bf7601f195589098ae57f)